main/Main.hs

```diff
@@ -316,16 +332,60 @@ listener dbUri dbChannel pool refConf refDbStructure mvarConnectionStatus connWo
 
 -- | Re-reads the config at runtime. Invoked on SIGUSR2.
 -- | If it panics(config path was changed, invalid setting), it'll show an error but won't kill the main thread.
+#ifndef mingw32_HOST_OS
 reReadConfig :: FilePath -> IORef AppConfig -> IO ()
 reReadConfig path refConf = do
   conf <- readValidateConfig path
   atomicWriteIORef refConf conf
   putStrLn ("Config file reloaded" :: Text)
+#endif
```

Comment:
I'm confused. Why did this not throw errors for the windows build before? What changed here?

Reply:
I explicitly exported only main in `module Main (main) where` at the top of Main.hs to make sure I didn't have any unused symbols - it didn't make a difference for my Linux builds, but on Windows it then rightfully complained about those symbols being unsused

Reply:
Ah, ok. Thanks for the explanation!